src/blocks/stars.css

```diff
@@ -0,0 +1,41 @@
+.card__container {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 24px;
+
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.card__star:nth-child(-n+4) {
```

Comment:
it should work not only with 4 as in the previous task

Reply:
isn't fixed